src/Pages/Homepage.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import fbIcon from '../static/images/icons8-facebook-48.png';
+import twitterIcon from '../static/images/icons8-twitter-48.png';
+import '../static/css/homepage.css';
+
+
+function Homepage() {
+  return (
+    <>
+      <header>
+        <div className="asd">
```

Comment:
more fitting class name?

Reply:
oops I was testing something and I forgot to change the name back 🤦